safe-global/web-core
# fix: adjust table styling + owner management

## What it solves

Table styling and owner management layout.

## How this PR fixes it

1. A new style override for table headers has been added that adjusts the typography to a `"body1"` variant with the `"secondary.light"` colour variant.
2. As per the Figma designes, the owner management table header now only shows "Name" and the actions have been reordered/are always shown.

## How to test it

1. Open the assets of a Safe. Observe the header text to be slightly larger and of a lighter colour than the table content.
2. Open the Safe setup settings. Observe the owner table to only have the "Name" header and all owner actions visible by default.

## Screenshots
![image](https://user-images.githubusercontent.com/20442784/186131962-1fa90ce1-80ee-42c1-97a5-974597f843b1.png)
![image](https://user-images.githubusercontent.com/20442784/186131982-3b210e77-5ef3-41af-a9b3-915cd05e0f55.png)
